ports/robin-map/portfile.cmake

```
@@ -1,15 +1,18 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO Tessil/robin-map
-    REF 622443f40544fb6a693402e69c1328d685eac939 # v0.6.3
-    SHA512 ec91ee3ce1a992cfc1b82c17d8d0d1d1af7078df9871b585526a81e1e05162b844958d3bf61badad191902d837832e29dc254755965410beac702ddf15c55a49
+    REF v0.6.3
+    SHA512 485557f300d33bda62bb8accdf246819ee8ffe956bc022e7ddca54ff6ad1a9fdb8db8d80690add3ef238e834d1eb8e2905920cb0a0674e7df010f6946d01297b
     HEAD_REF master
 )
 
-file(COPY ${SOURCE_PATH}/include/tsl DESTINATION ${CURRENT_PACKAGES_DIR}/include)
-
-file(INSTALL
-    ${SOURCE_PATH}/LICENSE
-    DESTINATION ${CURRENT_PACKAGES_DIR}/share/robin-map
-    RENAME copyright
+vcpkg_cmake_configure(
+    SOURCE_PATH ${SOURCE_PATH}
 )
+
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(CONFIG_PATH share/cmake/tsl-robin-map TARGET_PATH share/${PORT})
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH share/cmake/tsl-robin-map)
```
There is no `TARGET_PATH` argument in `vcpkg_cmake_config_fixup()`. Please see https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/ports/vcpkg-cmake-config/vcpkg_cmake_config_fixup.md

---

Sorry, I changed it to vcpkg_cmake_config_fixup without testing it. Fixed now